tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/ADAuthTokenProvider.cs

```diff
@@ -0,0 +1,78 @@
+﻿
+using Microsoft.Identity.Client;
+using Newtonsoft.Json.Linq;
+using UKHO.ERPFacade.API.FunctionalTests.FunctionalTests;
+
+namespace UKHO.ERPFacade.API.FunctionalTests.Helpers
+{
+    public class ADAuthTokenProvider
+    {
+        static string AzureADToken = null;
+        public static Config _config;
```

Comment:
This should be private scope, with an accessor, either a set accessor or an explicit SetConfig() method

Reply:
incorporated changes.